lib/ansible/plugins/lookup/conjur_variable.py

```diff
@@ -0,0 +1,163 @@
+# (c) 2018, Jason Vanderhoof <jason.vanderhoof@cyberark.com>, Oren Ben Meir <oren.benmeir@cyberark.com>
+# (c) 2018 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
```

Comment:
@jvanderhoof or @ryanprior,

could you add below here?

```
ANSIBLE_METADATA = {'metadata_version': '1.1',
                    'status': ['preview'],
                    'supported_by': 'community'}
```

Reply:
Added

Reply:
@thaumos - Thanks for the feedback. Mind giving it one more look? I'd love for this to be able to make in today if possible with code freeze on the 22nd.

Reply:
lgtm.  there shouldn't be a concern having this merged before code freeze.